src/hooks/useAccounts.tsx

```diff
@@ -178,7 +180,27 @@ const useAccountsContext = () => {
 
   useEffect(() => {
     (async () => {
-      if (connectedDydxAddress && signerGraz) {
+      if (walletType === WalletType.TestWallet) {
+        // Get override values. Use the testFlags value if it exists, otherwise use the previously
+        // saved value where possible. If neither exist, use a default garbage value.
+        const evmAddressOverride: EvmAddress = STRING_KEYS.TEST_WALLET as unknown as EvmAddress;
+        const addressOverride: string = testFlags.addressOverride ||
+          evmDerivedAddresses?.[evmAddressOverride]?.dydxAddress ||
+          'dydx1';
+
+        dispatch(setOnboardingState(OnboardingState.WalletConnected));
+
+        // Set variables.
+        saveEvmDerivedAccount({
+          evmAddress: evmAddressOverride,
+          dydxAddress: addressOverride as `dydx${string}`,
+        });
+        const wallet = new LocalWallet();
+        wallet.address = addressOverride;
+        setLocalDydxWallet(wallet);
+
+        dispatch(setOnboardingState(OnboardingState.AccountConnected));
```

Comment:
```suggestion
      if (walletType === WalletType.TestWallet) {
        // Get override values. Use the testFlags value if it exists, otherwise use the previously
        // saved value where possible. If neither exist, use a default garbage value.
        const addressOverride: string = testFlags.addressOverride;

        dispatch(setOnboardingState(OnboardingState.WalletConnected));

        const wallet = new LocalWallet();
        wallet.address = addressOverride;
        setLocalDydxWallet(wallet);

        dispatch(setOnboardingState(OnboardingState.AccountConnected));
```

I don't think we need to save the test address to local storage, since we will always be getting it from the testFlags instead of localstorage.

Reply:
Chatted with Brendan, this is for loading the address once `WalletConnectionType` is stored as `WalletType.TestWallet`